### ISSUE TITLE
The "data.win" context menu options are missing in Windows 11.

### ISSUE THREAD

**Issue by VladiStep** [UnderminersTeam/UndertaleModTool#1259]
### Describe the bug

It seems that Windows 11 changed how the file context menu works, so the context menu options like "Run game normally" don't appear, even the registry entries are there.

### Reproducing steps

1. Run UTMT at least once.
2. Right-click any "data.win" file.

### Setup Details

1. UTMT version - https://github.com/krzys-h/UndertaleModTool/commit/2a9315b21c265e11e39f14afdf9495409754b89b

**Comment by Miepee** [comment]
This isn't really a utmt issue. 
In win11, you have 2 right click menus: a mostly hardcoded win11 one with certain stuff that programs can't overwrite, and a "more options" that shows the old win10 right click menu.

**Comment by VladiStep** [comment]
Yeah, I know that it's still available in the "More options", but I think there is a way to add that to the new one

**Comment by Miepee** [comment]
I don't *think* there is, but feel free to research it.

**Comment by Miepee** [comment]
I'm not really interested in implementing this because it goes directly against win11's design philosophies. And i'd rather not go against OS philosophies.
